.eslintrc.js

```diff
@@ -20,10 +20,6 @@ module.exports = {
   ],
   plugins: ['prettier', 'jest', 'react'],
   rules: {
-    'prettier/prettier': [
```

Comment:
prefer to keep them here to match PF React... any reason you prefer them in package.json?

Reply:
we could extend them here later on as well..

Reply:
ok - i'm fine w/ removing them for now (and potentially reintroducing later...or not...hopefully). ;)

https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/443#discussion_r198525644